.github/workflows/publishbot.yml

```diff
@@ -27,11 +27,6 @@ jobs:
     name: 'Deploy the package'
     needs: npm-publish
     runs-on: [self-hosted, linux, x64, ec2-alpha]
-    container:
-      image: 458176070654.dkr.ecr.us-east-1.amazonaws.com/jenkins/deployment_package:stable
-      options: --user root
-      volumes:
-        - /var/run/docker.sock:/var/run/docker.sock
     steps:
       - uses: actions/checkout@v2
```

Comment:
Although we are no longer running a container, the checkout action does a `git clean`, so I don't expect old builds to be in the workspace.